pubspec.yaml

```diff
@@ -6,15 +6,15 @@ repository: https://github.com/vrtdev/flutter_workmanager
 issue_tracker: https://github.com/vrtdev/flutter_workmanager/issues
```

Comment:
Should it be changed to `issue_tracker: https://github.com/fluttercommunity/flutter_workmanager/issues`?
I know it works, but still

Reply:
Yes it should!